src/client/components/app/SummaryCard/index.tsx

```diff
@@ -10,9 +10,7 @@ const StyledCard = styled(Card)`
   position: relative;
 `;
 
-const StyledCardContent = styled(CardContent)<{ withArrow?: boolean }>`
-  ${({ withArrow }) => withArrow && `margin-right: 5rem`};
-`;
+const StyledCardContent = styled(CardContent)``;
```

Comment:
Use `CardContent` directly?

Reply:
Yes we can change this, we have some of these around the app since we were not against these wrappers in some cases or just vestiges of old refactors. I would create another PR to fix them all tho